spec/features/ncr/work_orders/requester_edit_spec.rb

```diff
@@ -7,7 +7,7 @@
     end
   end
 
-  let(:work_order) { create(:ncr_work_order, description: 'test') }
+  let(:work_order) { create(:ncr_work_order, org_code: Ncr::Organization.all[0].to_s,  description: 'test') }
```

Comment:
sure! I also like the idea of having a factory for this non-AR class since `all` loads all ~900 records into memory, which seems like overkill. will look into doing that and report back...